checker/tests/nullness/Issue1667.java

```diff
@@ -0,0 +1,21 @@
+// warning: Interpreting @com.sun.istack.internal.NotNull as a type annotation on an array component type.
```

Comment:
It would be much more user-friendly if these messages were attached to the first occurrence of the warning. That is, on lines 6 and 11 in this file instead of here without line numbers.

Reply:
The "first" occurrence might change depending on the order in which files are compiled or which files are compiled at once.  If the warning is suppressed relative to code location, the warning would not always be suppressed.

I'm thinking about making it suppressible using `-AsuppressWarnings`.